lib/slim_lint/linter/trailing_blank_lines.rb

```diff
@@ -0,0 +1,18 @@
+module SlimLint
+  # Checks for trailing whitespace.
+  class Linter::TrailingBlankLines < Linter
+    include LinterRegistry
+
+    on_start do |_sexp|
+      dummy_node = Struct.new(:line)
+
+      if !document.source.end_with?("\n")
+        report_lint(dummy_node.new(document.source_lines.size),
+                    'No blank line in the end of file')
+      elsif document.source.lines.last.blank?
+        report_lint(dummy_node.new(document.source.lines.size),
+                    'Multiple empty lines in the end of file')
```

Comment:
Sorry, I haven't understand question for the first time. This linters are different, and people may want to use them separately. Ex., they may want to disable EmptyLines to divide blocks with 2 or more empty lines, but still want to check to for single trailing blank line.